.github/release-please/release-please-config.json

```diff
@@ -3,6 +3,7 @@
     ".": {
       "release-type": "go",
       "bump-minor-pre-major": true,
+      "bump-patch-for-minor-pre-major": false,
```

Comment:
I reviewed this and, at least based on documentation, this should default to `false` anyway, but we can set it explicitly just in case. You can see the full config spec here: https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#configfile.

I still don't understand the issue we see in #1094, as the same "behavior" worked correctly in the past with the same configuration. See #954.